uptane/common.py

```diff
@@ -27,6 +44,17 @@ def sign_signable(signable, keys_to_sign_with):
     keys_to_sign_with:
       A list whose elements must conform to tuf.formats.ANYKEY_SCHEMA.
 
+    datatype:
```

Comment:
For the datatype argument?

The full text doesn't mentions the support datatypes.
```
datatype:
      The type of data signable['signed'] represents. This is necessary because
      if we're signing over ASN.1/DER, we need to convert to ASN.1/DER first,
      and conversion to ASN.1/DER varies by type. This value doesn't matter if
      signing is occuring over JSON.
```